### DISCUSSION TITLE
Support for LLM Studio

### DISCUSSION THREAD

**Discussion by nasimoo** [i-am-bee/beeai-framework#59] [Ideas]
Please add support for LLM Studio in Adapters

**Comment by Tomas2D** [comment]
Hello @nasimoo.

Issue created: https://github.com/i-am-bee/bee-agent-framework/issues/71

**Reply by streetcup** [reply]
And of course support for Ollama.

**Reply by Tomas2D** [reply]
Ollama is already supported.

**Reply by Tomas2D** [reply]
See https://github.com/i-am-bee/bee-agent-framework/blob/main/docs/llms.md
